course/test/src/isometry_TEST.cc

```diff
@@ -58,10 +95,19 @@ GTEST_TEST(Vector3Test, Vector3Operations) {
   t[1] = 2.;
   t.z() = 3.;
   EXPECT_EQ(t, p);
+
+  try
+  {
+    t[4] = 1;
+  }
+  catch (std::out_of_range const &err)
```

Comment:
Maybe this is some test that got forgotten in the code?

Reply:
Done